cql-source.kamelet.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: camel.apache.org/v1alpha1
+kind: Kamelet
+metadata:
+  name: cassandraql-source
```

Comment:
Do you plan to add other kind of cassandra sources? I think this can all be called cassandra-source. Look also at the file name that is different